express/code/blocks/pricing-table/pricing-table.css

```diff
@@ -545,6 +545,14 @@ main .section .pricing-table-wrapper:has(.pricing-table.many-cols) {
 
 
 @media (max-width: 980px) {
+  .pricing-table.seamless .row.table-end-row.section-row .col-1{
+    display: none;
+  }
+
+  .pricing-table.seamless .row.section-row .col {
+    border: none !important;
```

Comment:
can you use specificity to avoid?

Reply:
It doesn't look like you need the important at all. This selector is already more specific than the `border-width` declared on line 324. 

`!important` should be reserved only for very special use cases.

![image](https://github.com/user-attachments/assets/4b4be396-5e31-4582-91ce-11716415bf1b)